src/t8_schemes/t8_default/t8_dtet_bits.h

```diff
@@ -37,13 +37,24 @@ T8_EXTERN_C_BEGIN ();
 
 /** Compute the coordinates of a vertex of a tetrahedron.
  * \param [in] t    Input tetrahedron.
- * \param [out] coordinates An array of 2 t8_dtet_coord_t that
- * 		     will be filled with the coordinates of the vertex.
  * \param [in] vertex The number of the vertex.
+ * \param [out] coordinates An array of 3 t8_dtet_coord_t that
+ * 		     will be filled with the coordinates of the vertex.
  */
 void                t8_dtet_compute_coords (const t8_dtet_t * t, int vertex,
                                             t8_dtet_coord_t coordinates[3]);
```

Comment:
What is this function otherwise used for. Why is its name different from the element function?